### PR TITLE
fix polarity for concurrent replay

### PR DESCRIPTION
**Pull request by bw-solana**
#### Problem
Polarity is wrong for deciding whether to replay serially or concurrently.

I blame the reviewer of #137 

#### Summary of Changes
Switch polarity

### CODE REVIEW

File: core/src/replay_stage.rs [hunk 1]
```diff
@@ -654,7 +654,7 @@ impl ReplayStage {
                 )
             };
             // Thread pool to (maybe) replay multiple threads in parallel
-            let replay_mode = if replay_slots_concurrently {
```
Comment (steviez) [inline core/src/replay_stage.rs:657]: Can we switch the contents of the `if/else` cases to avoid the `!`.

Comment (bw-solana) [reply core/src/replay_stage.rs:657]: Anything for you

Comment (steviez) [reply core/src/replay_stage.rs:657]: Just trying to help bump up your line counts 😉

**Review by steviez — APPROVED** [review]
LGTM, thanks for catching this!

**Comment by steviez** [comment]
@bw-solana - I did also fix this in the other PR where a number is passed down instead of the bool:
https://github.com/anza-xyz/agave/pull/236/files#diff-6d8458bb2e53158ac472a9ad4709e6a0a52b75d930c019013298e8acda133828L657-R657

I believe that PR to be pretty close, just updated for all of Trent's comment so we could also just let that PR resolve this issue

**Comment by bw-solana** [comment]
> @bw-solana - I did also fix this in the other PR where a number is passed down instead of the bool: https://github.com/anza-xyz/agave/pull/236/files#diff-6d8458bb2e53158ac472a9ad4709e6a0a52b75d930c019013298e8acda133828L657-R657
> 
> I believe that PR to be pretty close, just updated for all of Trent's comment so we could also just let that PR resolve this issue

Yeah, either way works for me!

**Comment by steviez** [comment]
> Yeah, either way works for me!

In general, we shouldn't leave known bugs in master. At the same time, I'll probably have to do a (albeit an easy one) manual conflict resolve if we push this. How about we push this by end of day if my other PR hasn't shipped by then?

**Comment by bw-solana** [comment]
> How about we push this by end of day if my other PR hasn't shipped by then?

Done deal. Tick tock

**Pull request merged**